airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -197,22 +197,32 @@ def test_oneof_usage(self, actual_connector_spec: ConnectorSpecification):
             assert common_props, f"There should be at least one common property for {oneof_path} subobjects. {docs_msg}"
 
             const_common_props = set()
+            enum_common_props = set()
             for common_prop in common_props:
                 if all(["const" in variant["properties"][common_prop] for variant in variants]):
                     const_common_props.add(common_prop)
-            assert (
-                len(const_common_props) == 1
-            ), f"There should be exactly one common property with 'const' keyword for {oneof_path} subobjects. {docs_msg}"
+                if all(["enum" in variant["properties"][common_prop] for variant in variants]):
+                    enum_common_props.add(common_prop)
+            assert len(const_common_props) == 1 or (
+                len(const_common_props) == 0 and len(enum_common_props) == 1
+            ), f"There should be exactly one common property with 'const' keyword (or equivalent) for {oneof_path} subobjects. {docs_msg}"
```

Comment:
this relaxes the requirement for `len(const_common_props) == 1` by also allowing `len(enum_common_props) == 1` when `len(const_common_props) == 0`